src/rgw/rgw_s3select.cc

```diff
@@ -427,6 +427,9 @@ int RGWSelectObj_ObjStore_S3::run_s3select_on_csv(const char* query, const char*
     csv.use_header_info=true;
   }
 
+  if(m_outputFormat == OutputFormat::JSON) {
```

Comment:
nit: `if` is not a function, put a space before the opening parenthesis, please.